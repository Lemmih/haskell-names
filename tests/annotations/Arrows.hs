{-# LANGUAGE Arrows #-}
module Arrows where

x y = proc z -> do
  a <- y -< z
  b <- y -< a
  y -< b
