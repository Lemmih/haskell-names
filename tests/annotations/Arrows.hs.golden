y        at  5:8 is a local value defined at 4:3
z        at 5:13 is a local value defined at 4:12
y        at  6:8 is a local value defined at 4:3
a        at 6:13 is a local value defined at 5:3
y        at  7:3 is a local value defined at 4:3
b        at  7:8 is a local value defined at 6:3
